File: VB/04_Shapes/ApplyAnimationOnChart.vb
Imports Spire.Presentation
Imports Spire.Presentation.Charts
Imports Spire.Presentation.Drawing.Animation

Namespace ApplyAnimationOnChart
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create an instance of presentation document
			Dim presentation As New Presentation()
			presentation.LoadFromFile("..\..\..\..\..\..\..\Data\AnimationChart.pptx")
			'Get the first slide
			Dim slide As ISlide = presentation.Slides(0)
			'Get chart
			Dim shape As IShape = slide.Shapes(0)
			If TypeOf shape Is IChart Then
				'Apply Wipe animation effect to the chart
				Dim effect As AnimationEffect = slide.Timeline.MainSequence.AddEffect(shape, AnimationEffectType.Wipe)
				'Set the BuildType as Series
				effect.GraphicAnimation.BuildType = GraphicBuildType.BuildAsSeries
			End If

			'Save the document
			Dim result As String = "ApplyAnimationOnChart.pptx"
			presentation.SaveToFile(result, FileFormat.Pptx2013)

			'Launch the document
			PresentationDocViewer(result)
		End Sub

		Private Sub PresentationDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace

File: CS/04_Shapes/ApplyAnimationOnChart.cs
using System;
using System.Windows.Forms;
using Spire.Presentation;
using Spire.Presentation.Charts;
using Spire.Presentation.Drawing.Animation;

namespace ApplyAnimationOnChart
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create an instance of presentation document
            Presentation presentation = new Presentation();
            presentation.LoadFromFile(@"..\..\..\..\..\..\..\Data\AnimationChart.pptx");
            //Get the first slide
            ISlide slide = presentation.Slides[0];
            //Get chart
            IShape shape = slide.Shapes[0];
            if (shape is IChart)
            { 
                //Apply Wipe animation effect to the chart
                AnimationEffect effect = slide.Timeline.MainSequence.AddEffect(shape, AnimationEffectType.Wipe);
                //Set the BuildType as Series
                effect.GraphicAnimation.BuildType = GraphicBuildType.BuildAsSeries;
            }
           
            //Save the document
            String result = "ApplyAnimationOnChart.pptx";
            presentation.SaveToFile(result, FileFormat.Pptx2013);

            //Launch the document
            PresentationDocViewer(result);
		}
	
		private void PresentationDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}